portfolio.html

```diff
@@ -4,8 +4,42 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="brand">
+        <h1 class="brand-name">Brand Name</h1>
+      </div>
+      <div class="hamburger-btn">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
```

Comment:
This content should be wrapped in a `<nav>` tag.

<img width="379" alt="Screen Shot 2022-08-18 at 8 38 57" src="https://user-images.githubusercontent.com/82426682/185409071-66257165-1779-4be9-8507-f57fbd8631a0.png">

Please fix it.